src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -87,6 +87,7 @@
 #define	XMC_HOST_MSG_OFFSET_REG		0x300
 #define	XMC_HOST_MSG_ERROR_REG		0x304
 #define	XMC_HOST_MSG_HEADER_REG		0x308
+#define	XMC_STATUS2_REG			0x30C
```

Comment:
Strange mix and match of spaces and tabs :-(

Reply:
Could you please point out where is strange? after #define, there is a tab.

Reply:
There is an alignment problem and the following uses some spaces...